OneCommunityGlobal/HighestGoodNetworkApp
# Nahiyan_Dark mode on dropdown items

# Description
Dark mode is implemented on all dropdown items on the header

## Related PRS (if any):
This frontend PR is related to the dev backend

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner
5. go to any page besides the dashboard or timelog page (They seem to be broken in development)
6. Once you can see the header, select on reports, other links and welcome dropdowns.
7. verify dark mode is working properly both in light mode and dark

## Screenshots or videos of changes:
![Screenshot 2024-06-06 211226](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/2194b029-2e17-4174-86e4-9742b845d078)
![Screenshot 2024-06-06 211243](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/f43cda37-603b-4dda-ae0c-cf0af54df00b)
